packages/instant-apps-components/src/components/instant-apps-app-guide/instant-apps-app-guide.tsx

```diff
@@ -1,11 +1,16 @@
 import { Component, Host, h, Prop, State } from '@stencil/core';
-import { Element, HostElement } from '@stencil/core/internal';
+import { Element, HostElement, Watch } from '@stencil/core/internal';
 import AppGuideViewModel from './AppGuide/AppGuideViewModel';
 import { AppGuidePage, AppGuideRenderType } from './AppGuide/interfaces/interfaces';
 import AppGuide_T9n from '../../assets/t9n/instant-apps-app-guide/resources.json';
 import { getMessages } from '../../utils/locale';
 import { ArrowType } from '@esri/calcite-components';
 
+const CSS = {
+  contentList: 'esri-content-list',
```

Comment:
Can these class names be updated to `instant-apps-app-guide__x`? To follow naming conventions of class names of other components

Reply:
Done